lake/writer.go

```diff
@@ -190,17 +184,3 @@ func (s *ImportStats) Copy() ImportStats {
 		RecordsWritten:     atomic.LoadInt64(&s.RecordsWritten),
 	}
 }
-
-func importComparator(zctx *zed.Context, pool *Pool) *expr.Comparator {
-	layout := pool.Layout
-	layout.Keys = field.List{poolKey(layout)}
-	return zbuf.NewComparator(zctx, layout)
-}
-
-func poolKey(layout order.Layout) field.Path {
-	if len(layout.Keys) != 0 {
-		// XXX We don't yet handle multiple pool keys.
-		return layout.Keys[0]
-	}
-	return field.New("ts")
```

Comment:
We should make this change (dropping "ts" as the default), but in a separate PR.

Reply:
Ok.